src/requisitionMerge.js

```diff
@@ -110,11 +110,6 @@ const minimalOutgoingLine = outgoingLine => ({
   requiredItem: !outgoingLine.skipped,
 });
 
-const getConsumption = incomingLine => {
```

Comment:
Not sure what's going on here. Just ignore the duplicate changes from #143, think it's due to a force push due to a git config issue.